app/src/main/res/values/strings.xml

```diff
@@ -2,16 +2,16 @@
 <resources>
 
     <!-- Applications list -->
-    <string name="info_applications_list_refreshed">Applications list refreshed</string>
-    <string name="button_refresh_list">Refresh the applications list</string>
-    <string name="error_application_not_found">The application %1$s has not been found!</string>
+    <string name="info_applications_list_refreshed">App list refreshed</string>
+    <string name="button_refresh_list">Refresh app list</string>
+    <string name="error_application_not_found">Could not find the %1$s app.</string>
 
     <!-- Favorites -->
-    <string name="info_no_favorites_yet">You can add your favorites applications here\nby clicking the menu button.</string>
-    <string name="button_manage_favorites">Manage favorites</string>

```

Comment:
Things like "Bookmarks" is always "Bookmarks" (?), following this logic, "Favorites" shouldn't include "Manage".

Reply:
Not sure I want to change that one for now. Your point is clear and I see what you mean, but on the other hand I also like the menu entries starting with action verbs like now.

Reply:
I think one entry in strings_settings is missing the actionable/operative verb.
Will look at it later, brain spent for now.

Reply:
There is indeed the last entry "Settings / Help" for which it felt strange with a verb (tried some at the beginning but I was not very satisfied with the result, and since it is the last entry I decided to let it like that).

Reply:
After further thinking, it is ok for me to change this entry menu to "Favorites" only.
I will adapt the other entries accordingly ("Folders" and "Hidden apps").